### PR TITLE
EASEᴿ Implementation

### PR DESCRIPTION
**Pull request by yilmazerhakan**
### Checklist:

- [x] I have updated the documentation accordingly.
- [x] I have updated `README.md`.
- [x] I have updated `examples/ease_movielens.py` (if you are adding a new example).

### CODE REVIEW

File: cornac/models/ease/recom_ease.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        When False, the model is not trained and Cornac assumes that the model is already \
+        trained. 
+        
+    verbose: boolean, optional, default: False
+        When True, some running logs are displayed.
+
+    seed: int, optional, default: None
+        Random seed for parameters initialization.
+
+    init_par: numpy 1d array, optional, default: None
```
Comment (tqtg) [inline cornac/models/ease/recom_ease.py:30]: Should it be removed?

File: cornac/models/ease/recom_ease.py [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        Returns
+        -------
+        self : object
+        """
+        Recommender.fit(self, train_set, val_set)
+
+        # A rating matrix
+        self.U = self.train_set.matrix
+
+        # Grahm matrix is X^t X, compute dot product
```
Comment (tqtg) [inline cornac/models/ease/recom_ease.py:79]: Gram matrix?

File: docs/source/models.rst [hunk 3]
```diff
@@ -232,3 +232,8 @@ Weighted Matrix Factorization (WMF)
 --------------------------------------------------
 .. automodule:: cornac.models.wmf.recom_wmf
    :members:
+
+Embarrassingly Shallow Autoencoders for Sparse Data (EASEᴿ)
```
Comment (tqtg) [inline docs/source/models.rst:236]: This should be added after the AMR model. The list is organized chronologically.

**Comment by tqtg** [comment]
@yilmazerhakan thanks for the great contribution! If no further changes, I'll merge this PR soon.

**Comment by yilmazerhakan** [comment]
Thank you very much, I will be honor if I contribute to this beautiful framework.

**Pull request merged**